okd/src/build.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/bash
+
+
+# replace microshift assets to Upsteam from OKD
+okd_url=quay.io/okd/scos-release
+okd_releaseTag=4.17.0-0.okd-scos-2024-08-21-100712
+
+oc adm release info ${okd_url}:${okd_releaseTag} >all_images
+
+for op in $(cat assets/release/release-x86_64.json  | jq -e -r  '.images | keys []') 
+do
+    image=$(oc adm release info --image-for=${op} ${okd_url}:${okd_releaseTag} || true) 
+    if [ -n "${image}" ] ; then
+        echo "${op} ${image}"
+        jq --arg a "${op}" --arg b "${image}"  '.images[$a] = $b' assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp
+
+
+        # delete openssl image from assets - just to verify if we still need it,since it doesnt referenced anywhere
+        jq '. | del (.images["openssl"])'  assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp2
+
+        mv /tmp/release-x86_64.json.tmp2 assets/release/release-x86_64.json
+    fi
+done
+
+
+sudo podman stop microshift-okd
+rm -rf okd/src/rpmbuild
+rm -rf _output/rpmbuild
+make build 
+make rpm
+createrepo _output/rpmbuild/
+cp -rf _output/rpmbuild microshift-okd/src/
+
+# build the container
+sudo podman build -f okd/src/microshift-okd-source.containerfile -t microshift-okd
```

Comment:
I'm still not sure what's the difference between two containerfiles and when should they be used? They seems totally unrelated.

Reply:
well its two different approaches , so we can decide:
1. microshift-okd-source.containerfile -   when building  your microshift RPM on the dev Machine , so it requires installing bunch of pre-req (gcc,linker,make,go ) on the dev.so  during container build-time the RPMs copied ..
2.  microshift-okd-multi-build.Containerfile - RPM build  done in container using multi-stage dockerfile.

the second approach is more community friendly , since it only requires a container engine , so i vote for it.

Reply:
I think keeping only multi-build Containerfile is better - it's a complete standalone file that can be build on any host